### PR TITLE
docs: Add missing UI font settings to "Configuring Zed"

### PR DESCRIPTION
**Pull request by jsparkdev**
- Add missing `ui_font` options in `configuring-zed.md`

Release Notes:

- N/A

### CODE REVIEW

File: docs/src/configuring-zed.md [hunk 1]
```diff
@@ -2178,6 +2178,64 @@ Float values between `0.0` and `0.9`, where:
 }
 ```
 
+## UI Font Family
+
+- Description: The name of a font to use for rendering in the UI.
+- Setting: `ui_font_family`
+- Default: `Zed Plex Sans`
```
Comment (maxdeviant) [inline docs/src/configuring-zed.md:2185]: This isn't the default UI font anymore.

Comment (notpeter) [reply docs/src/configuring-zed.md:2185]: It's not?
https://github.com/zed-industries/zed/blob/2470db490115322720f85e2ad57e922de930ee1b/assets/settings/default.json#L56

Comment (maxdeviant) [reply docs/src/configuring-zed.md:2185]: Ah right, I was searching for "IBM" in the name.

Forgot we still had our own fork 🤦🏻‍♂️

**Review by maxdeviant — APPROVED** [review]
Thanks!

**Pull request merged**